config/crd/bases/kuadrant.io_dnspolicies.yaml

```diff
@@ -155,6 +155,19 @@ spec:
                         type: integer
                     type: object
                 type: object
+              providerRef:
```

Comment:
wondering do we want to use a new version here as we are adding quite a few changes that are not compatible with the current version?

Reply:
What you thinking? Move dnspolicy from v1alpha1 to v1alpha2 with no conversion (Drop support for v1alpha1). I was kind of hoping since it was still an alpha API we were OK with jut changing it, but i guess moving it to v1alpha2 isn't a huge issue.

Reply:
yes drop support for v1alpha1 so no conversion needed. Reason for changing is it communicates a big change a little better. I was on the fence a little but think getting into this process would be good